client/src/components/row/Row.js

```diff
@@ -1,17 +1,51 @@
 import './Row.css';
 
+
+
 function Row(props) { 
+    
+    if (props.displayX) {
+        return doDisplayX(props);
+    } else {
+        return dontDisplayX(props);
+    }
+}
+
+function doDisplayX(props) {
     return (
         <div
             className="proposal-box"
             id={props.title}
             onClick={() => {props.changeTitle(props.title);
                             props.changeDescription(props.description);}
         }>
+            <div className="whole-row">
+                <div className="leftRow">
+                    <img src={props.x} className="x-box"></img>
+                </div>
+                <div className="rightRow">
+                    <div className="proposal-title">{props.title}</div>
+                    <img src={props.vote} className="vote-status"></img>
+                </div>
+            </div>
+        </div>
+    );
+}
 
+function dontDisplayX(props) {
```

Comment:
can we make the names of these functions more accurate? For instance, having something like "rowDeletionMode" or "rowViewingMode" makes it so that if, in the future, we get rid of the x's (or change them to checkboxes), these function names still accurately describe the intent :)